keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -68,10 +69,10 @@ class SqlDeliveryConfigRepository(
         )
         .from(DELIVERY_CONFIG)
         .where(DELIVERY_CONFIG.APPLICATION.eq(application))
-        .fetch { (uid, name, application, serviceAccount) ->
+        .fetchOne { (uid, name, application, serviceAccount) ->
```

Comment:
why it was changed from `fetch` to `fetchOne`? Is it possible to have more than one?

Reply:
It's not - I removed the ability to have more than one. If there is it'll be a database failure. So This is just updating this method call to reflect that.